### DISCUSSION TITLE
Drawer and Mobile browsers

### DISCUSSION THREAD

**Discussion by zerolivesleft** [saadeghi/daisyui#716] [Q&A]
Hi, I love the look and functionality of DaisyUI. 

I'm currently using the navbar that when viewed on mobile coverts to a hamburger menu/drawer. 

It works great but one problem im having is since everything is wrapped in a single fixed div, the html element doesn't scroll causing the ui on mobile browsers to not collapse/minimize. Is there a way around this?

**Comment by saadeghi** [comment]
If you mean the dynamic parts of mobile browsers (addressbar for example), there's no way to fix that as far as I know. They only collapse when the body scrolls ☹️

**Reply by zerolivesleft** [reply]
Thank you for getting back to me so quickly!

Yes, that's what I'm referring to. I've been able to do a few workarounds but I always end up back to how you have it in your example, as everything else tends to have other side effects that are much worse.

Is it possible to pull the content out of the wrapper and still have the drawer work? I haven't been successful, so I'm sure I'd need to change some CSS at the very least.

**Reply by saadeghi** [reply]
An alternative would be using modal: https://play.tailwindcss.com/DCcZD7ISDg

You can also use Tailwind CSS utilities to make your own sidebar (But you would need to handle responsive and toggling the sidebar yourself): 
https://play.tailwindcss.com/KIvnVBnpMI
